tee-worker/omni-executor/Makefile

```diff
@@ -79,6 +79,10 @@ start-local:
 stop-local:
 	docker compose -f $(OMNI_DIR)/docker/docker-compose.yml down
 
-.PHONY: get-pallet-metadata
-get-pallet-metadata:
-	subxt metadata --url http://localhost:9944 --allow-insecure --pallets OmniAccount,Teebag > parentchain/artifacts/rococo-omni-account.scale
+.PHONY: get-metadata
+get-metadata:
+	subxt metadata --url http://localhost:9944 --allow-insecure --pallets OmniAccount,Teebag > parentchain/api-interface/artifacts/metadata.scale
+
+.PHONY: generate-api-interface
```

Comment:
should this target depend on `get-metadata` ? (always get updated `metadata.scale` ?)

Reply:
Yep 👍🏼